website/content/en/preview/troubleshooting.md

```diff
@@ -139,3 +139,36 @@ Provisioners created without those tags and run in more recent Karpenter version
 ```
 If you are providing a [custom launch template]({{<ref "./aws/launch-templates" >}}), specifiying a `subnetSelector` is still required.
 However, specifying a `securityGroupSelector` will cause a validation error.
+
+## Pods stuck in "Pending" state for up to 30 minutes before transitioning to "Running"
```

Comment:
Can we include the `Security Groups for Pods` in the heading?  `Pods using Security Groups for Pods stuck in "Pending" state for up to 30 minutes before transitioning to "Running"`

Reply:
sure.